c42SharedLibScripts/c42SharedLibrary.py

```diff
@@ -35,9 +35,9 @@ class c42Lib(object):
 	#cp_password = "<pw>"
 
 	# Test values
-	cp_host = "http://localhost"
+	cp_host = "https://demo.code42.com"
```

Comment:
please leave as localhost for simplicity and keeping the shared library as un-specific as possible